WordPress/src/main/java/org/wordpress/android/ui/posts/services/AztecImageLoader.java

```diff
@@ -76,12 +78,16 @@ public void onResponse(ImageLoader.ImageContainer response, boolean isImmediate)
                     final String cacheKey = url + maxWidth;
                     // Make sure both width ad height respect the max size for the editor
                     bitmap = ImageUtils.getScaledBitmapAtLongestSide(bitmap, maxWidth);
-                    WordPress.getBitmapCache().putBitmap(cacheKey, bitmap);
-                    // By default, BitmapFactory.decodeFile sets the bitmap's density to the device default so, we need
-                    // to correctly set the input density to 160 ourselves.
-                    bitmap.setDensity(DisplayMetrics.DENSITY_DEFAULT);
-                    BitmapDrawable bitmapDrawable = new BitmapDrawable(context.getResources(), bitmap);
-                    callbacks.onImageLoaded(bitmapDrawable);
+                    if (bitmap != null) {
```

Comment:
AS says this is always `true`. I've verified and getScaledBitmapAtLongestSide` returns the original bitmap (that is not null) if the resize is not possible.